spec/spec_helper.rb

```diff
@@ -21,8 +21,10 @@
 
 # Requires factories defined in lib/solidus_paypal_commerce_platform/testing_support/factories.rb
 require 'solidus_paypal_commerce_platform/testing_support/factories'
+require 'support/script_tag_helper'
```

Comment:
This is probably not required because it's already requiring every file and folder in `spec/support`.